app/model/SubscriptionData.scala

```diff
@@ -69,23 +70,23 @@ case class PaperData(
   deliveryInstructions: Option[String],
   plan: CatalogPlan.Paper
 ) {
-  val sanitizedDeliveryInstructions = deliveryInstructions.map(instructions => instructions.replaceAll("\"", ""))
+  val sanitizedDeliveryInstructions: Option[String] = deliveryInstructions.map(instructions => instructions.replaceAll("\"", ""))
 }
 
 object PersonalData {
-  def fromIdUser(u: IdUser) = {
+  def fromIdUser(u: IdUser): PersonalData = {
     val phoneNumber: Option[String] = for {
-      identityPhoneNumber <- u.privateFields.flatMap(_.telephoneNumber)
+      identityPhoneNumber <- u.privateFields.telephoneNumber
       countryCode <- identityPhoneNumber.countryCode
       localNumber <- identityPhoneNumber.localNumber
     } yield NormalisedTelephoneNumber(countryCode, localNumber).format
 
     val personalData = PersonalData(
-      title = u.privateFields.flatMap(_.title).flatMap(Title.fromString),
-      first = u.privateFields.flatMap(_.firstName).mkString,
-      last = u.privateFields.flatMap(_.secondName).mkString,
+      title = u.privateFields.title.flatMap(Title.fromString),
+      first = u.privateFields.firstName.mkString,
+      last = u.privateFields.secondName.mkString,
       email = u.primaryEmailAddress,
-      receiveGnmMarketing = u.statusFields.flatMap(_.receiveGnmMarketing).getOrElse(false),
+      receiveGnmMarketing = false,
```

Comment:
just to clarify, this change is related to the whole thing about the field that we're reading from isn't actually set from anywhere anyway, so it was already effectively hard coded to false

Reply:
It's also deprecated, and unused - it's not a GDPR-compliant consent. So in theory, it should be null, or defaulted within the case class constructor.

Reply:
@johnduffell yeah - correct and I meant to put a comment but forgot. Put I think Paul's point now takes precedent, so I'll add that instead.